### PR TITLE
chore(sample): add Graph Connector sample

### PR DESCRIPTION
**Pull request by formulahendry**
image

image

### CODE REVIEW

File: packages/fx-core/src/common/samples-config.json [hunk 1]
```diff
@@ -162,6 +162,16 @@
...
             "suggested": false
+        },
+        {
+            "id": "graph-connector-app",
+            "title": "Graph Connector App",
+            "shortDescription": "Showcases how to build custom Graph Connector",
+            "fullDescription": "This sample app showcases how to build custom Graph Connector with Azure Functions and query data using Microsoft Graph Client and TeamsFx SDK.",
+            "tags": ["Tab", "Azure function", "TS", "SSO"],
+            "time": "10min to run",
+            "configuration": "Ready for local debug",
```
Comment (MuyangAmigo) [inline packages/fx-core/src/common/samples-config.json:173]: Is it ready for local debug? I think it needs manual steps to consent the permission first?

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (formulahendry) [reply packages/fx-core/src/common/samples-config.json]: Updated:
image

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Comment by blackchoey** [comment]
We may need to optimize the size of gif. Since it seems larger than other samples. Large gif size increases VS Code extension size, there was a bug previously for this.

**Comment by HuihuiWu-Microsoft** [comment]
> We may need to optimize the size of gif. Since it seems larger than other samples. Large gif size increases VS Code extension size, there was a bug previously for this.

Suggested size is 1600*920, you can check whether your gif has correct size.

**Comment by formulahendry** [comment]
> We may need to optimize the size of gif. Since it seems larger than other samples. Large gif size increases VS Code extension size, there was a bug previously for this.

@blackchoey Try to minify the size to 1MB.

**Review by blackchoey — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**